src/components/Switch/Switch.js

```diff
@@ -111,6 +94,38 @@ class Switch extends React.PureComponent {
   }
 }
 
-export default React.forwardRef((props, ref) => (
-  <Switch innerRef={ref} {...props} />
+const basePropTypes = {
+  className: PropTypes.string,
+  defaultOn: PropTypes.bool,
+  name: PropTypes.string,
+  on: PropTypes.bool,
+  onChange: PropTypes.func,
+  size: PropTypes.oneOf(acceptedSizes)
+};
+
+/* eslint-disable react/default-props-match-prop-types */
+const baseDefaultProps = {
+  defaultOn: false,
+  onChange: noop,
+  size: 'basic',
+  name: baseClass
+};
+
+SwitchComponent.propTypes = {
+  ...basePropTypes,
+  innerRef: PropTypes.instanceOf(
+    typeof Element === 'undefined' ? () => {} : Element
```

Comment:
@dprzybylek take a look at https://github.com/livechat/design-system/pull/43
We need to figure out a better way to avoid SSR issues.